src/main/java/controllers/ConsoleInputController.java

```diff
@@ -221,6 +222,46 @@ public void onCommandReceived(String input) {
                 consoleView.consolePrint("Project created!");
             }
             break;
+        case "manage":
+            if (inputReader.hasNext()) {
+                int projectNumber = Integer.parseInt(inputReader.next());
+                IProject projectToManage = projectRepository.getProject(projectNumber);
+                String projectName = projectToManage.getDescription();
+                consoleView.consolePrint("Now managing: " + projectName);
+                //branch into project and begin to read commands to manage project
+                Scanner manageProjectInput = new Scanner(System.in);
```

Comment:
Are we able to remove the number of new Scanner objects?

Reply:
I created a new scanner because we need to branch into a new project and read new commands separate from the main line of commands. If I abstract out into a new controller as you suggested, would that fix the problem?

Reply:
Hmm not really either. It's okay! Let's keep it for now, just KIV in the future